service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -419,14 +425,14 @@ private void checkSumIfCourierLimitBySumOfOrder(TariffsInfo tariffsInfo, Long su
     public FondyOrderResponse saveFullOrderToDB(OrderResponseDto dto, String uuid, Long orderId) {
         final User currentUser = userRepository.findByUuid(uuid);
         TariffsInfo tariffsInfo = tryToFindTariffsInfoByBagIds(getBagIds(dto.getBags()), dto.getLocationId());
-        Map<Integer, Integer> amountOfBagsOrderedMap = new HashMap<>();
+        List<OrderBag> bagsOrdered = new ArrayList<>();
```

Comment:
please move the creation of the list inside formBagsToBeSavedAndCalculateOrderSum